Singer.API/DTOs/Csv/CsvRegistrationDTO.cs

```diff
@@ -0,0 +1,36 @@
+using CsvHelper.Configuration;
+using Singer.Models;
+using Singer.Resources;
+using System;
+
+namespace Singer.DTOs.Csv
+{
+   public class CsvRegistrationDTO
+   {
+      public string CaseNumber { get; set; }
+      public string FirstName { get; set; }
+      public string LastName { get; set; }
+      public AgeGroup AgeGroup { get; set; }
+      public bool IsExtern { get; set; }
+      public DateTime? DayCareBeforeStartDateTime { get; set; }
+      public DateTime? DayCareAfterEndDateTime { get; set; }
+      public RegistrationStatus Status { get; set; }
+      public string LegalGuardianName { get; set; }
+
+      public class Mapper : ClassMap<CsvRegistrationDTO>
+      {
+         public Mapper()
+         {
+            Map(x => x.CaseNumber).Index(0).Name(DisplayNames.CaseNumber);
```

Comment:
Why don't we use automapper here?

Reply:
Because this mapper is not for db, but for csv. It is a way to ensure the properties are in a certain order and a certain the name, format,.. is used during the serialization